web/src/components/RunDetailTrigger/RunDetailTrigger.styled.ts

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+  display: flex;
+  height: 100%;
+  width: 100%;
+`;
+
+export const Section = styled.div`
+  flex: 1;
+`;
+
+export const SectionLeft = styled(Section)`
+  background-color: ${({theme}) => theme.color.white};
+  z-index: 1;
+`;
+
```

Comment:
Do we need to add scroll here too? I saw your video and it seems that the scroll is applied to the page and not the sections.